config/recipes/beats/heartbeat_es_kb_health.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: heartbeat
+spec:
+  type: heartbeat
+  version: 7.8.0
+  elasticsearchRef:
+    name: elasticsearch
+  kibanaRef:
+    name: kibana
+  config:
+    heartbeat.monitors:
+    - type: tcp
+      schedule: '@every 5s'
+      hosts: ["elasticsearch-es-http.default.svc:9200"]
+    - type: tcp
+      schedule: '@every 5s'
+      hosts: ["kibana-kb-http.default.svc:5601"]
+  deployment:
+    replicas: 1
+    podTemplate:
+      spec:
+        dnsPolicy: ClusterFirstWithHostNet
```

Comment:
Same here for heartbeat, I don't see the advantage of running in the host network and would break in clusters where the nodes can't communicate with the pods (albeit a rare situation). I think a comment justifying it makes sense